src/response/chi0.jl

```diff
@@ -291,49 +292,59 @@ Perform in-place computations of the derivatives of the wave functions by solvin
 a Sternheimer equation for each `k`-points. It is assumed the passed `δψ` are initialised
 to zero.
 """
-function compute_δψ!(δψ, basis, H, ψ, εF, ε, δHψ; ψ_extra=[zeros_like(ψk, size(ψk,1), 0) for ψk in ψ],
-                     kwargs_sternheimer...)
+function compute_δψ!(δψ, basis::PlaneWaveBasis{T}, H, ψ, εF, ε, δHψ, ε_minus_q=ε;
+                     ψ_extra=[zeros_like(ψk, size(ψk,1), 0) for ψk in ψ],
+                     q=zero(Vec3{T}), kwargs_sternheimer...) where {T}
+    # We solve the Sternheimer equation
+    #   (H_k - ε_{n,k-q}) δψ_{n,k} = - (1 - P_{k}) δHψ_{n, k-q},
```

Comment:
What is `ε_{n,k-q}` ?  Ok, reading on I begin to understand. Maybe have a subparagraph in the general comments around line 194 making clear what is the difference between a q=0 and a q\neq 0 calculation. Otherwise this is the first thing one reads and the jargon and context has not yet been made clear.